integration_tests/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "integration_tests",
+  "version": "1.0.0",
+  "description": "",
+  "main": "http-errors.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "datadog-lambda-js": "^3.27.0"
+  },
+  "devDependencies": {
+    "serverless-plugin-datadog": "^2.2.1"
```

Comment:
NIT: Explicitly set the version of the plugin to avoid changes in the plugin to break these integration tests